device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-C256S2/th5-a7060x6-64pe.config.bcm

```diff
@@ -1926,3 +1926,12 @@ device:
         DEVICE_CONFIG:
             AUTOLOAD_BOARD_SETTINGS: 0
 ...
+---
+device:
+    0:
+        TM_THD_CONFIG:
+            SKIP_BUFFER_RESERVATION: 0
+            THRESHOLD_MODE: LOSSY
+        TM_SCHEDULER_CONFIG:
+            NUM_MC_Q: NUM_MC_Q_2
+...
```

Comment:
The BCM file seems to be missing all the queue configurations, this might cause problems. even CPU queue is not here, not sure if this is expected.

Reply:
service pool is also missing, is this expected? (ideally, this can be done by buffer SAI APIs, but in the 256x200G HWSKU, it didn't really work well.)

Reply:
This is for pure lossy config using SDK defaults. Will need to check if explicit service pool configs are required.